### PR TITLE
Evacuation can prevent xeno major

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Change the evacuation system:
- Add more evacuation shuttles on most maps
- The pods launch 10 minutes after the evacuation order, rather than 3 currently
- If more than 50% of marines manage to evacuate, xeno cannot get a major (they still can get minor if they kill the rest of the marines)

## Why It's Good For The Game

This gives marines another option in hijack situation :
- They can try to guard SD for 20 minutes, and the round is draw
- They can try to wipe out xenos and get a minor
- Or now, they can try to evacuate and deny xenos a major. Usefull if marines are really not enough to win or even to hold SD

## Changelog
:cl:
add: Evacuation can prevent xeno major
/:cl:

### CODE REVIEW

File: code/modules/shuttle/escape_pod.dm [hunk 1]
```diff
@@ -176,98 +170,6 @@
 		M.cryopods += src
 		linked_to_shuttle = TRUE
 
-/obj/machinery/cryopod/evacuation/ex_act(severity)
```
Comment (BraveMole) [inline code/modules/shuttle/escape_pod.dm:179]: I'm fixing this code in the cryo PR

**Comment by Losenis** [comment]
We might need a more reliable way for the evac door to stay open all the way until evac, sometimes it remains shut all throughout or closes at seemingly random points with zero way to open it.

**Comment by Lumipharon** [comment]
How does this work exactly? Is it 50% of surviving marines at the time of evac, evacing successfully? Or 50% based off when benos landed or something?

Basically could ungas suicide into the xenos if they can't get to evac, to try deny xenos the major?

**Comment by bowlofcereal** [comment]
Only a coward retreats if the situation is remotely salvageable.

**Comment by novaepee** [comment]
This is way better than marines retreating #5400 before xenomorphs can hijack.

**Comment by Walarks** [comment]
> This is way better than marines retreating #5400 before xenomorphs can hijack.

I don't think this is better or good overall since it gives marines more incentive to early evac to or plan for shipside rather than hold groundside.

**Comment by TiviPlus** [comment]
This probably made doors crush people and close on them

**Review by TiviPlus — APPROVED** [review]

**Comment by Derrick2Electricboogaloo** [comment]
So is the shuttle limit bigger, or is it still arbitrarily 10 or something close?

**Pull request merged**